cluster/gce/gci/configure-helper.sh

```diff
@@ -805,7 +805,7 @@ contexts:
   name: webhook
 EOF
   fi
-  if [[ "${ENABLE_EGRESS_VIA_KONNECTIVITY_SERVICE:-false}" == "true" ]]; then
+  if [[ "${ENABLE_EGRESS_VIA_KONNECTIVITY_SERVICE:-false}" == "grpc" ]]; then
```

Comment:
this seems like a hack but ... sure :-)

Reply:
not going to block this PR on this, but for literal strings prefer `'grpc'`, single quoted contents stay literal.

Reply:
got rid of the hack :-)